### PR TITLE
Import initial code

### PR DESCRIPTION
**Pull request by bessbd**
This change does the initial code import.

### CODE REVIEW

**Review by ricellis — CHANGES_REQUESTED** [review]
Need the pom version updates. There are some model files here that shouldn't be as well, but I think we'll clean those up later.

File: modules/cloudant/pom.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    <artifactId>cloudant</artifactId>
+    <packaging>jar</packaging>
+    <name>Cloudant Generated</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.ibm.cloud</groupId>
+            <artifactId>cloudant-common</artifactId>
+            <version>0.0.3</version>
```
[the remark this answers is not in the document]
Comment (bessbd) [reply modules/cloudant/pom.xml]: Done in 81b328d

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,482 @@
...
+    </scm>
+
+    <issueManagement>
+        <system>Github</system>
+        <url>https://github.com/IBM/cloudant-java-sdk/issues</url>
+    </issueManagement>
+
+    <ciManagement>
+        <system>Travis-CI</system>
+        <url>https://travis.ibm.com/cloudant/cloudant-java-sdk/</url>
```
Comment (emlaver) [inline pom.xml:87]:
```suggestion
        <url>https://travis-ci.com/IBM/cloudant-java-sdk/</url>
```

**Review by ricellis — APPROVED** [review]

**Comment by bessbd** [comment]
Thank you for the review, @emlaver and @ricellis . I'm about to merge this.

**Pull request merged**